website/docs/workloads/mlperf/mlperf.md

```diff
@@ -1,40 +1,67 @@
-﻿# MLPerf
-MLPerf 2.0 is a consortium of AI leaders from academia, research labs, and industry whose mission is to “build fair and useful benchmarks” that provide unbiased evaluations of training and inference performance for hardware, software, and services—all conducted under prescribed conditions. To stay on the cutting edge of industry trends, MLPerf continues to evolve, holding new tests at regular intervals and adding new workloads that represent the state of the art in AI.
-* [MLPerf Training Documentation](https://github.com/mlcommons/training_results_v2.0/blob/main/MLPerf%E2%84%A2%20Training%20v2.0%20Results%20Discussion.pdf)  
+# MLPerf
+
+MLPerf is a consortium of AI leaders from academia, research labs, and industry whose mission is to “build fair and useful benchmarks” that provide unbiased evaluations of training and inference performance for hardware, software, and services—all conducted under prescribed conditions. To stay on the cutting edge of industry trends, MLPerf continues to evolve, holding new tests at regular intervals and adding new workloads that represent the state of the art in AI.
+
+* [MLPerf Training Documentation](https://github.com/mlcommons/training_results_v2.1/blob/main/MLPerf%E2%84%A2%20Training%20v2.0%20Results%20Discussion.pdf)  
 * [MLPerf Inference Documentation](https://github.com/mlcommons/inference_results_v2.0)  
-* [MLPerf Training Benchmarks](https://github.com/mlcommons/training_results_v2.0/tree/main/NVIDIA/benchmarks)
+* [MLPerf Training Benchmarks](https://github.com/mlcommons/training_results_v2.1/tree/main/NVIDIA/benchmarks)
 * [MLPerf Inference Benchmarks](https://github.com/mlcommons/inference_results_v2.0/tree/master/closed/NVIDIA)
+* [MLPerf Training Bert Preprocessing Data](./mlperf-trainining-bert-preprocessing-data.md)
 
 ## System Requirements
+
 This is a GPU-specific workload and requires high-performance graphic cards to run. It is recommended that the system-under-test have a high-performing Nvidia (e.g. M60 or higher) or AMD (e.g. MI25 or higher)
 graphics card.
 
 ## Supported Hardware Systems
-The following section defines the hardware systems/SKUs on which the MLPerf 2.0 workload will run effectively in cloud environments. These hardware systems contain
+
+The following section defines the hardware systems/SKUs on which the MLPerf workload will run effectively in cloud environments. These hardware systems contain
 GPU components for which the MLPerf workload is designed to test.
 
-* **Datacenter systems**  
+* **Datacenter systems MLPerf Inference**  
   * A100-SXM-80GBx8 (NVIDIA DGX A100, 80GB variant)
-  * A100-SXM-80GBx4 (NVIDIA DGX Station A100, "Red October", 80GB variant)
+  * A100-SXM-80GBx4 (NVIDIA DGX Station A100, "Red SEPTober", 80GB variant)
   * A100-PCIex8 (80GB variant)
   * A2x2
   * A30x8
 
-* **Edge Systems**
+* **Edge Systems MLPerf Inference**
```

Comment:
sorry why is there a distinction between datacenter and edge? do we need to list them separately?

Reply:
https://github.com/mlcommons/inference_results_v2.0/tree/master/closed/NVIDIA

These are mentioned on original page of inference we can ask in ml commons community as comment for MlPerf Inference 
![image](https://github.com/microsoft/VirtualClient/assets/29044654/e7d7ccaf-6bb7-47c0-b5fc-361f89b4b503)